src/rocksdb/src/lib.rs

```diff
@@ -530,6 +530,15 @@ fn rocksdb_core_loop<K, V, M, O, IM>(
     let retry_max_duration = tuning_config.retry_max_duration;
     let rocksdb_options = options.as_rocksdb_options(&tuning_config);
 
+    if let Some(buf) = tuning_config.shared_write_buffer_manager.get() {
+        tracing::info!(
+            "RocksDB using WriteBufferManager {{ enabled: {:?}, buffer_size: {:?}, memory_usage: {:?} }}",
+            buf.enabled(), buf.buffer_size(), buf.memory_usage()
+        );
+    } else {
+        tracing::info!("RocksDB not using WriteBufferManager");
+    }
+
```

Comment:
lets move this to `new` and also add the `path` we opened it at

Reply:
ack

Reply:
The write buffer manager is initialized with `options.as_rocksdb_options` before this line. Which `new` did you mean here?

Reply:
(chatted offline)